source/Playnite.DesktopApp/ViewModels/GameEditViewModel.cs

```diff
@@ -921,6 +925,15 @@ List<Guid> consolidateIds(SelectableDbItemList selectionList, List<Guid> origina
                     game.OverrideInstallState = EditingGame.OverrideInstallState;
                 }
 
+                if (appSettings.GameSortingNameAutofill && string.IsNullOrEmpty(game.SortingName))

```

Comment:
Also this should be testing against `EditingGame` not `game`, since user might have changed sorting name during this edit session. Actually it will need to test against both in case this is multi game edit.